migrations/20201019001536-create-organization_applications.ts

```diff
@@ -0,0 +1,41 @@
+import Base from 'db-migrate-base';
+import { promisify } from 'util';
+/**
+ * Describe what your `up` migration does.
+ */
+export function up(db: Base, callback: Base.CallbackFunction): void {
+  db.runSql(
+    `
+      CREATE TABLE OrganizationApplications
+        (
+          id                   SERIAL,
+          status               status NOT NULL,
+          organization_id      INTEGER NOT NULL,
+          question             VARCHAR(255) NOT NULL,
+          PRIMARY KEY (id),
+          FOREIGN KEY (organization_id) references organizations(id)
```

Comment:
This looks good, but let's add `UNIQUE (organization_id)` before this to make it a one to one relationship